avocado/utils/configure_network.py

```diff
@@ -132,10 +133,7 @@ def set_mtu_host(interface, mtu):
     except process.CmdError as ex:
         raise NWException("MTU size can not be set: %s" % ex)
     try:
-        cmd = "ip add show %s" % interface
-        mtuvalue = process.system_output(cmd, shell=True).decode("utf-8") \
-                                                         .split()[4]
-        if mtuvalue == mtu:
+        if mtu in genio.read_file("/sys/class/net/%s/mtu" % interface) and is_interface_link_up(interface):
```

Comment:
Why here, you are using `genio.read_file` and on another method, you are using `self.session.cmd`? If possible, let's try to use the same method.

Reply:
ok i will change, genio.read_file i couldn't able to use in remote machine s i did like that.